kaisen/kaisen/src/raliat/function.py

```diff
@@ -0,0 +1,10 @@
+def subtract(a, b):
+    return a - b
+
+def add(a, b):
+    difference = subtract(a, b)
+    return a + b, difference
+
+result, difference = add(10, 5)
+print(f"Addition result: {result}")
+print(f"Subtraction result: {difference}")
```

Comment:
Hi Raliat, we don't need this `function.py` file. the other`p.py` file one works ok!